test/System.IdentityModel.Tokens.Jwt.Tests/JwtSecurityTokenHandlerTests.cs

```diff
@@ -40,6 +40,75 @@ namespace System.IdentityModel.Tokens.Jwt.Tests
 {
     public class JwtSecurityTokenHandlerTests
     {
+        [Theory, MemberData(nameof(VerifyValidatedTokenTheoryData))]
+        public void VerifyValidatedToken(JwtTheoryData theoryData)
+        {
+            var context = new CompareContext();
+            SecurityToken validatedToken = null;
+            try
+            {
+                var claimsIdentity = theoryData.TokenHandler.ValidateToken(theoryData.Token, theoryData.ValidationParameters, out validatedToken).Identity as ClaimsIdentity;
+                Assert.True(validatedToken != null);
+                theoryData.ExpectedException.ProcessNoException(context);
+            }
+            catch (Exception ex)
+            {
+                Assert.True(validatedToken == null);
+                theoryData.ExpectedException.ProcessException(ex, context.Diffs);
+            }
+
+            TestUtilities.AssertFailIfErrors(context);
+        }
+
+        public static TheoryData<JwtTheoryData> VerifyValidatedTokenTheoryData
+        {
+            get
+            {
+                var theoryData = new TheoryData<JwtTheoryData>();
+                var handler = new JwtSecurityTokenHandler();
+
+                ClaimsIdentity claimsIdentity = new ClaimsIdentity(ClaimSets.DefaultClaimsIdentity);
+                var validationParameters = Default.AsymmetricSignTokenValidationParameters;
+                validationParameters.ValidateIssuer = true;
```

Comment:
Why is this assignment important? Isn't this true by default?

Reply:
Yes, ValidateIssuer is true by default.

I added it with the below reference in mind. Assuming that the below is set to make the test case behavior easy to understand.
Eg: ExpectedException for JwtTheory data is NoExceptionExpected by default but it is still assigned in few places.

Reply:
Got it, thanks.